### ISSUE TITLE
[Enhancement]: Button for a car mode please

### ISSUE THREAD

**Issue by K3R0-42** [advplyr/audiobookshelf#3345]
### Type of Enhancement

Web Interface/Frontend

### Describe the Feature/Enhancement

Larger buttons for operation in the car (Tesla Browser). Some other apps solve this by allowing you to click on the cover of the album you are listening to so that you have a player to operate. I have sent some pictures of a cell phone app to illustrate this. 

### Why would this be helpful?

I use the software in the Tesla, as there is no app for it I have to open it in the browser which only takes up half the screen when driving. Unfortunately, the buttons for operating the software are very small, which has often caused me to jump to the wrong chapter in the audio book instead of pressing pause as intended. It would therefore be good to have the option of larger control buttons. 

### Future Implementation (Screenshot)

![IMG_4966](https://github.com/user-attachments/assets/dafe33c2-51b4-4256-89db-845b580c9250)
![IMG_4969](https://github.com/user-attachments/assets/b6c2d923-124c-45fe-ba92-01cdf9f85757)
![IMG_4968](https://github.com/user-attachments/assets/657ab84d-7949-4b6d-a1be-1207153e0c6f)

### Audiobookshelf Server Version

v2.12.3

**Comment by nichwall** [comment]
Related to https://github.com/advplyr/audiobookshelf/issues/2569

**Comment by advplyr** [comment]
BookPlayer is a mobile app though and you are using the web client. The Abs mobile app also allows opening a full screen player.

I think a full screen player on the web client at full width might be strange.

**Comment by K3R0-42** [comment]
When I click on the cover in the web player, a full screen of the cover also opens, so it would certainly be possible to place buttons there for easier operation.

**Comment by suzhoum** [comment]
Unrelated to this issue, but when I use ABS on Tesla Browser, it can't seem to automatically start the next episode if I minimized the Browser App. It only starts the next episode when I reopen the Browser. Does it happen to any of you?

**Comment by K3R0-42** [comment]
I have the “problem” too, but I'm glad that it works with a minimized browser. At least it's enough to watch something quickly without it stopping playback immediately. Before the update, minimizing didn't work at all.

**Comment by Eroskuro** [comment]
> I think a full screen player on the web client at full width might be strange.

I have to disagree with this. Sure if you are only using the webui in a browser on a desktop or laptop to get the book playing it's fine, but there are time where a full screen (or at least a full page) player would be useful. Some examples would be on devices where there is not a mobile app, or on a HTPC. There are other scenarios that I could think of but tbh all are a BIT niche.

If there was at least a option for it that would be nice.

**Comment by Madj42** [comment]
I would like to see the bigger buttons capability brought in for this.  I recently moved to a Tesla from Android Auto and it is a huge pain to click around with the small text and buttons.

**Comment by oransterf** [comment]
This would be awesome!

**Comment by lajustman** [comment]
I will add my name to the "want" list.
